CMakeLists.txt

```
@@ -15,6 +15,7 @@ include_directories(
 
 if(WIN32)
   add_compile_options(/wd4996)
+  configure_file(libxml-2.0.pc.in ${PROJECT_SOURCE_DIR}/libxml-2.0.pc)
```

```suggestion
  add_compile_definitions(LIBXML_STATIC)
```

Link errors are because libxml2 is build as static library but try lo link to symbols used in dynamic library.

See:  https://stackoverflow.com/questions/75807248/unresolved-external-symbols-from-libxml-functions-when-compiling-podofo-in-visua

This probably fixes the two issues: build failure at update to 2.11.7 and build failure on UWP-64 when build multiple add-ons at same time (pvr.stalker not included in Xbox installer).

---

Thanks added and working!